### ISSUE TITLE
Edit Page V2: Allow User to Choose Content Editing for Individual or All Pages

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#27551]
### Parent
#27546 

### User Story
As a content editor, I want to be informed if the content I am editing on a page will affect other pages, so that I can make an informed decision about whether to edit the content for all pages or only for the current page.

<img width="739" alt="image" src="https://github.com/dotCMS/core/assets/751424/36a764b0-a024-4495-bfbc-a5d3572b67d8">

```[tasklist]
### Acceptance Criteria
- [x] When the user clicks on the "Edit" button for a content item on a page, a dialog should be displayed.
- [x] Dialog only shows if the edit content is being used in another page
- [x] The functionality should be consistent with the existing "Edit" functionality in the old edit page.
- [x] The feature should be implemented in the new edit page.
- [x] This functionality should work for headless pages as well
```

### Proposed Objective
Static content: "Core Features"

### Proposed Priority
Static content: "Priority 3 - Average"

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.
Static content: "N/A"

**Comment by rjvelazco** [comment]
### QA Team

This issue is waiting for this one: https://github.com/dotCMS/core/issues/27658

**Comment by josemejias11** [comment]
**Failed**: Tested on **master_f3b48ba**, Docker, macOS 13.0, FF v121.0.1

According with @rjvelazco the property needs to be restore with the use of the new sdk

**Comment by rjvelazco** [comment]
@josemejias11 Fix: https://github.com/dotCMS/core/pull/28030

**Comment by josemejias11** [comment]
**Approved**: Tested on **dotcms-qa-master.dotcms**, Docker, macOS 13.0, FF v121.0.1

**Comment by weiyiwang-dotcms** [comment]
Hi, I have a support case https://dotcms.zendesk.com/agent/tickets/115969 related to this. Customer is on v 24.04.05, and is using EMA. They are not receiving the popup when they click on the edit button, even though the content is on multiple pages. 
[screen-capture-edit-modal-popup-not-displayed.webm](https://github.com/dotCMS/core/assets/136110669/9576f8ea-6aed-4d9f-9b56-f7d2b623d76b)

**Comment by josemejias11** [comment]
Hi @weiyiwang-dotcms, 

Currently this functionality is working properly on https://dotcms-qa-master.dotcms.site/c which is the environment where EMA was configured for testing. 

<img width="635" alt="Screenshot 2024-04-16 at 2 29 21 PM" src="https://github.com/dotCMS/core/assets/129882031/7231224e-6489-4521-928a-0838314ef8cb">


@rjvelazco can you please check on this on why this is and confirm if is a version issue or something else. Thank you so much!

**Comment by rjvelazco** [comment]
It might be due to the `SDK` being out of synchrony. We had issues publishing the `SDK` on that version. Do we want to port it in `release-24.04.05`? @weiyiwang-dotcms @fmontes @josemejias11

**Comment by weiyiwang-dotcms** [comment]
Hi, the issue is still being reported by the customer. Here is a video that captures the issue.
[screen-capture-modal-popup-still-not-displayed.webm](https://github.com/dotCMS/core/assets/136110669/e366b054-8a6d-4b62-9ab2-6f1dd2faa1e4)

**Comment by bryanboza** [comment]
I think here the problem reported by @weiyiwang-dotcms is in the old editor, this card is related to the new editor. Then we can close this one for now.

**Comment by weiyiwang-dotcms** [comment]
In what version of dotCMS does the new editor become available? Is the new editor available in release-24.04.16?

**Comment by rjvelazco** [comment]
@nollymar Should we close this card and open a new one for the old editor to check the case @weiyiwang-dotcms reported?
